tests/qcmp.rs

```diff
@@ -21,21 +21,37 @@ use tokio::time::Duration;
 use quilkin::{protocol::Protocol, test_utils::TestHelper};
 
 #[tokio::test]
-async fn ping() {
+async fn proxy_ping() {
     let mut t = TestHelper::default();
-    let server_port = 12348;
+    let server_port = quilkin::test_utils::available_addr().await.port();
     let server_proxy = quilkin::cli::Proxy {
         port: server_port,
         to: vec![(Ipv4Addr::UNSPECIFIED, 0).into()],
         ..<_>::default()
     };
     let server_config = std::sync::Arc::new(quilkin::Config::default());
     t.run_server(server_config, server_proxy, None);
+    ping(server_port).await;
+}
+
+#[tokio::test]
+async fn agent_ping() {
+    let qcmp_port = quilkin::test_utils::available_addr().await.port();
+    let agent = quilkin::cli::Agent {
+        qcmp_port,
+        ..<_>::default()
+    };
+    let server_config = std::sync::Arc::new(quilkin::Config::default());
+    let (_, rx) = tokio::sync::watch::channel(());
+    tokio::spawn(async move { agent.run(server_config, rx).await });
```

Comment:
```suggestion
    tokio::spawn(async move { agent.run(server_config, rx).await.expect("Agent should run") });
```

This showed me the issue that:

```
called `Result::unwrap()` on an `Err` value: Elapsed(())
thread 'agent_ping' panicked at 'Agones should run: Address already in use (os error 98)

Location:
    /home/mark/workspace/quilkin/src/cli/agent.rs:98:26', tests/qcmp.rs:50:14
stack backtrace:
```

Doesn't look like qcmp is starting.